src/sagemaker/utils.py

```diff
@@ -1553,20 +1553,23 @@ def safe_members(members):
 
     for file_info in members:
         if _is_bad_path(file_info.name, base):
-            print(stderr, file_info.name, "is blocked (illegal path)")
+            logger.error(stderr, file_info.name, "is blocked (illegal path)")
```

Comment:
Have we checked the error log in console via unit test?
I have not seen this log format so hoping to confirm if it can print out the message as expected.
Usually we use format like :
```
logger.debug(
                    "Selecting TrainingInput's input_mode (%s) for TrainingInputMode.",
                    inputs.config["InputMode"],
                )
```
Ref: https://github.com/aws/sagemaker-python-sdk/blob/3aa0a56f7b369110c96acd4507310d221e48d3ce/src/sagemaker/estimator.py#L2491-L2494